deno_dist/helper/ssg/index.ts

```diff
@@ -78,17 +78,41 @@ export const fetchRoutesContent = async <
   const baseURL = 'http://localhost'
 
   for (const route of inspectRoutes(app)) {
-    if (route.isMiddleware) continue
+    if (route.isMiddleware || isDynamicRoute(route.path)) continue
 
-    const url = new URL(route.path, baseURL).toString()
+    const contentData = await fetchRouteContent(app, route.path, baseURL)
+    if (contentData) {
+      htmlMap.set(route.path, contentData)
+    }
+  }
+
+  return htmlMap
+}
+
+interface RouteContent {
+  content: string | ArrayBuffer
+  mimeType: string
+}
+
+const fetchRouteContent = async <E extends Env = Env, S extends Schema = {}>(
```

Comment:
Should you include these `fetchRouteContent` things in this PR?

Reply:
I don't need it, so I reverted it.